home/models.py

```diff
@@ -90,11 +91,18 @@ def get_paragraph_values(self):
         paragraph_values = self._get_child_block_values('paragraph')
         return '\n'.join(paragraph_values)
 
+    def get_absolute_url(self):
```

Comment:
not sure why this is necessary

Reply:
The `comments_xtd` library uses the `et_absolute_url` function of the model to redirect to after every successful action.

Reply:
Move this to the Mixin